File: src/main/java/com/distribution/modules/api/pojo/vo/DisMemberInfoAmountsVO.java
package com.distribution.modules.api.pojo.vo;

import com.distribution.modules.account.entity.MemberAccount;
import com.distribution.modules.dis.entity.DisMemberInfoEntity;
import lombok.Data;

import java.math.BigDecimal;

@Data
public class DisMemberInfoAmountsVO {

    private DisMemberInfoEntity disMemberInfoEntity;
    private MemberAccount memberAccount;
    //提现
    private BigDecimal withdrawalAmount;
    //总收入
    private BigDecimal generalIncome;
}

File: src/main/java/com/distribution/modules/api/pojo/vo/DisMemberInfoAmountsVO.kt
package com.distribution.modules.api.pojo.vo

import com.distribution.modules.account.entity.MemberAccount
import com.distribution.modules.dis.entity.DisMemberInfoEntity
import java.math.BigDecimal


data class DisMemberInfoAmountsVO(

        var disMemberInfoEntity: DisMemberInfoEntity = DisMemberInfoEntity(),

        var memberAccount: MemberAccount = MemberAccount(),

        //提现
        var withdrawalAmount: BigDecimal = BigDecimal.ZERO,

        //总收入
        var generalIncome: BigDecimal = BigDecimal.ZERO

)
